stackslib/src/chainstate/nakamoto/miner.rs

```diff
@@ -485,16 +485,6 @@ impl NakamotoBlockBuilder {
 
         let ts_end = get_epoch_time_ms();
 
-        if let Some(observer) = event_observer {
```

Comment:
Curious why this is getting dropped?

Reply:
It's invoked by the caller so that it can include the miner's signature.